### YamlMime:UniversalReference
items:
  - uid: azure-servicefabric.ValidationFailedChaosEvent
    name: ValidationFailedChaosEvent
    fullName: ValidationFailedChaosEvent
    children:
      - azure-servicefabric.ValidationFailedChaosEvent.kind
      - azure-servicefabric.ValidationFailedChaosEvent.reason
      - azure-servicefabric.ValidationFailedChaosEvent.timeStampUtc
    langs:
      - typeScript
    type: interface
    summary: 在验证过程与失败相对应的混沌测试事件。
    extends:
      name: azure-servicefabric.ChaosEvent
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.kind
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: 多态鉴别器
    syntax:
      content: 'kind: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.reason
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: 介绍为什么生成 ValidationFailedChaosEvent。 因为多个 MaxPercentUnhealthyNodes MaxClusterStabilizationTimeout 超过不正常，则可能发生这种情况。 此原因将作为字符串 ValidationFailedChaosEvent 的原因属性中。
    optional: true
    syntax:
      content: 'reason?: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.timeStampUtc
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: UTC 时间戳时生成该混沌测试事件。
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: azure-servicefabric
references:
  - uid: azure-servicefabric.ChaosEvent
    name: ChaosEvent
    spec.typeScript:
      - name: ChaosEvent
        fullName: ChaosEvent
        uid: azure-servicefabric.ChaosEvent